_data/releases.yml

```diff
@@ -1,66 +1,73 @@
 - date: 2024-03-01
   codebase: "CppInterOp"
   version: "v1.2.0"
+  description: "This release includes some improvements and bug fixes."
   link: "https://github.com/compiler-research/CppInterOp/releases/tag/v1.2.0"
 
 - date: 2024-02-29
   codebase: "Clad"
   version: "v1.4"
+  description: "Forward Mode & Reverse Mode: Improve handling of char and string literals. Reverse Mode: Add support for differentiating switch statements, supportpassing pointers as call arguments, and Fix pointer arithmetic for array types."
```

Comment:
there are recent major performance enhancements that the release notes should capture.